### DISCUSSION TITLE
DATA LOSS - creating an issue produces an error

### DISCUSSION THREAD

**Discussion by thany** [community/community#41013] [Projects and Issues]
### Select Topic Area

Bug

### Body

When I create an issue, it says on the next page:

> You can't perform that action at this time.

And my input is gone. Good thing I copied it to my clipboard, innit.

So,

# This is a form of DATA-LOSS

And should therefor be considered critical priority.

**Comment by jfullstackdev** [comment]
that happens from time to time for so many reasons, 

try it in Incognito mode, try changing ISP or use VPN, clear cache in regular tab, check how much RAM do you have, check how slow is your Internet connection, check whether you have GitHub extensions installed

**Reply by thany** [reply]
That doesn't do away the fact that this is a data loss issue, and should be fixed with the highest priority.

One solution could be to keep the data in localStorage, so that going to the new issue page again doesn't present an empty set of fields.

**Reply by thany** [reply]
But seriously, those are your suggestions? Then how am I able to work with Github completely fine, being able to do everything else totally fine? Only creating an issue is broken.

**Reply by jfullstackdev** [reply]
never have that issue in company development setting,

you still have your local commit, say when you lost it going to GitHub, you can commit it again, but I doubt the exact term is data loss since it was not lost, it was just unavailable during that time 

and never had that experience in personal and company development setting for several years and even other developers in my company never had that issue either
